DSA-Campaign-Uplift-Estimation/src/main/hangout_bot/datastore.py

```diff
@@ -0,0 +1,254 @@
+from google.cloud import datastore
+from campaigndata import CampaignData
+from userdata import *
+from constant import *
+
+# [Datastore Interaction Functions]
+
```

Comment:
nit: inconsistent number of new lines between functions

Reply:
Fixed